source/_components/vacuum.xiaomi_miio.markdown

```diff
@@ -312,3 +382,21 @@ Steps to take:
 6. Go to `\apps\com.xiaomi.smarthome\db`.
 7. Open `miio2.db` with the SQLite Browser.
 8. You can find your device tokens in `devicerecord` table.
+=======
+1. Install an old Version of MiHome (e.g. Mi-Home version 5.0.30) on your Android-Device
+2. Open MiHome, log-in and add your devices
+3. Enable USB-Debugging on your Android
+4. Create a backup from your MiHome App, by using adb
+  ```bash
+  adb backup com.xiaomi.smarthome
+  ```
+  Now the backup App opens on you Android-Device. You don't need to set a password, just click save.
+5. Extract the backup-file with android-backup-extractor
+  ```bash
+  java -jar abe.jar unpack backup.ab backup.tar
+  ```
+  After that, you kann open the file with WinRaR or what ever you like.
+6. Go to \apps\com.xiaomi.smarthome\db
+7. Open miio2.db with SQLite Browser
+8. You can find your device tokens in "devicerecord" table
+>>>>>>> rytilahti tips
```

Comment:
Same as above here, remove the remnants of the merge. Btw, why not simply point to python-miio's documentation (and improve them, if necessary): https://python-miio.readthedocs.io/en/latest/discovery.html#tokens-from-mi-home-logs ?

Reply:
This section is not mine..